uktrade/data-hub-frontend
# feature/cls2-489-refactor-item-renderers

## Description of change

- Add a new ItemRenderer component that handles the shared logic when displaying reminders, such as displaying deleted buttons and item headings.
- Use the new component by the other item renderers to reduce the amount of duplicated code
- Move the item renderers into appropriate subfolders

## Test instructions

Make sure you have some reminders assigned to you, and go to the http://localhost:3000/reminders page. All the existing pages should continue to work the same

## Screenshots

N/A

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
